### DISCUSSION TITLE
Make failing after upgrade to 22.10

### DISCUSSION THREAD

**Discussion by Dave-catcher** [BOINC/boinc#4988] [Q&A] [answered]
Getting the following at end of Make on master downloaded yesterday evening. Compiles without errors if I do just the client without manager.

```
/tmp/ccxMKXwA.s: Assembler messages:
/tmp/ccxMKXwA.s:32745: Error: unknown .loc sub-directive `vieu'
/tmp/ccxMKXwA.s:32745: Error: unknown pseudo-op: `.lvu7621'
/tmp/ccxMKXwA.s:354848: Error: leb128 operand is an undefined symbol: .LVU7621
/tmp/ccxMKXwA.s:354849: Error: leb128 operand is an undefined symbol: .LVU7621
make[2]: *** [Makefile:1636: boincmgr-SkinManager.o] Error 1
```

**Comment by AenBleidd** [comment, marked as answer]
I think you have a broken gcc installation. Please check if these help you:
- https://bugzilla.redhat.com/show_bug.cgi?id=1552529
- https://bbs.archlinux.org/viewtopic.php?id=245354

**Reply by Dave-catcher** [reply]
I re-installed gcc but also removed the --optimise flag at the ./configure stage so not quite sure which worked but now compiles correctly.  Many thanks.
